### ISSUE TITLE
Support "custom expected message"

### ISSUE THREAD

**Issue by constructor-igor** [constructor-igor/NUnit.That.Resharper.Plugin#12]
added test "executionCustomerMessage.cs"

```c#
        [Test]
        [ExpectedException(typeof(NotImplementedException), ExpectedMessage = "customer message")]
        public void TestExpectedExceptionWithExpressions()
        {
            {caret}foo4("customer message");
        }
```

Debugger shows next:

foundAttribute.GetText()	"ExpectedException(typeof(NotImplementedException), **ExpectedMessage = \"customer message\"**)"	string

but number of arguments: foundAttribute.Arguments.Count	**1**	int

argument [0] shows first argument only:
		foundAttribute.Arguments[0].GetText()	"typeof(NotImplementedException)"	string

Found unexpected resharper behavior:
Attribute.GetText() shows expected message:

![image](https://cloud.githubusercontent.com/assets/1849690/6828567/33c793e6-d317-11e4-8a86-a674501abff7.png)

Attribute shows 1 arguments only:
![image](https://cloud.githubusercontent.com/assets/1849690/6828590/53e3230c-d317-11e4-88bb-0c995c9c2052.png)

Argument [0] shows expected exception type:
![image](https://cloud.githubusercontent.com/assets/1849690/6828605/66081ca4-d317-11e4-90b2-16b58ff83fc4.png)

**Comment by kropp** [comment]
`ExpectedMessage` is not an argument, it's a property. Try `foundAttribute.PropertyAssignments` instead.

**Comment by constructor-igor** [comment]
Yes, PropertyAssignments contains expected message:
		**foundAttribute.PropertyAssignments[0].GetText()	"ExpectedMessage = \"customer message\""	string**

**Comment by constructor-igor** [comment]
.NameIdentifier.GetText() = "ExpectedMessage"
.Source.GetText() = "\"customer message\""

**Comment by constructor-igor** [comment]
implemented in 0.7.0.8-alpha
